src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -172,62 +172,62 @@ private String getDropdownMenuLabel(FeedbackQuestionAttributes question) {
     }
 
     private boolean isVisibilitySetToAnonymousToRecipientAndInstructors(FeedbackQuestionAttributes question) {
-        boolean responsesVisibleOnlyToRecipientAndInstructors = question.showResponsesTo.size() == 2
+        boolean isResponsesVisibleOnlyToRecipientAndInstructors = question.showResponsesTo.size() == 2
```

Comment:
This is now a grammatical error due to the plural `responses`. I feel the rule should include `are` and `have` as the plural counterparts to `is` and `has`. Thoughts?